core/sdk-core/src/main/java/software/amazon/awssdk/core/traits/Trait.java

```diff
@@ -16,6 +16,7 @@
 package software.amazon.awssdk.core.traits;
 
 import software.amazon.awssdk.annotations.SdkProtectedApi;
+import software.amazon.awssdk.core.SdkField;
```

Comment:
Why are we adding that import?

Reply:
`SdkField` is being used as a link in the javadocs. Without the import javadoc does not know how to resolve the type and generate the link.